### PR TITLE
TC Coverage for Various APIs

### PR DESCRIPTION
**Pull request by lokeshbv**
Expands and addsTC for fs,  network, pthread, termios, kernel, libc APIs

### CODE REVIEW

File: apps/examples/testcase/le_tc/kernel/tc_libc_stdlib.c [hunk 1]
```diff
@@ -46,24 +46,35 @@
...
-* @description          :Function for tc_libc_stdlib_qsort
-* @return               :int
-*/
+ *@fn					:thread_func
+ *@description			:Function for tc_libc_stdlib_abort
+ *@return				:void*
+ */
+static void *thread_func(void *arg)
+{
+	abort();
```
Comment (pradeepns85) [inline apps/examples/testcase/le_tc/kernel/tc_libc_stdlib.c]: Isn't this test case stop execution after calling abort ? How to continue the test case after this test case

File: apps/examples/testcase/le_tc/kernel/tc_pthread.c [hunk 2]
```diff
@@ -1345,6 +1345,9 @@ static void tc_pthread_pthread_sigmask(void)
 	sigemptyset(&st_newmask);
 	sigaddset(&st_newmask, SIGQUIT);
 
+	ret_chk = pthread_sigmask(333, &st_newmask, &st_oldmask);
```
Comment (pradeepns85) [inline apps/examples/testcase/le_tc/kernel/tc_pthread.c]: make a #define for 333

**Comment by pradeepns85** [comment]
Kindly check the below comments
1)  Modify the pull request header 
2)  Edit the commit description to reflect the changes
3)  Take care of other comments inside src files.

**Comment by lokeshbv** [comment]
@pradeepns85 ,Will take care of comments and will open new pull request.

**Pull request closed without merging**